lib/plugin/gitlab/GitLab.js

```diff
@@ -17,15 +18,14 @@ class GitLab extends Release {
     this.assets = [];
     const { certificateAuthorityFile, secure } = this.options;
 
-    const httpsOptions = {
-      certificateAuthority: certificateAuthorityFile ? fs.readFileSync(certificateAuthorityFile) : undefined,
-      rejectUnauthorized: typeof secure === 'boolean' ? secure : undefined
-    };
-
-    // Remove keys with undefined values
-    const https = _.pickBy(httpsOptions, value => value !== undefined);
-
-    this.certificateAuthorityOption = _.isEmpty(https) ? {} : { https };
+    this.certificateAuthorityOption = {
```

Comment:
Is it possible to have `this.certificateAuthorityOption` be an empty object as to minimize risks for existing users than don't need the custom `Agent`?

Reply:
yep, no problems, I'll update it